.github/workflows/deploy.yml

```diff
@@ -76,26 +110,68 @@ jobs:
             services+=("\"webhook\"")
           fi
 
+          # Parse service secrets and generate deploy_matrix
+          for service in "${services[@]}"; do
```

Comment:
I am curious if there is a better way to construct this JSON using an inline node script or maybe zx from Google. This bash string interpolation is easy to understand but it's also super weird.

Reply:
Sure, I'll improve in later PR as it not pressing enough.